### DISCUSSION TITLE
Watched status sync?

### DISCUSSION THREAD

**Discussion by jaredcat** [croneter/PlexKodiConnect#1957] [Q&A] [answered]
Does watching stuff in Kodi mark it as watched in Plex?

If not, does anyone have a working solution for synching what episodes/movies have been watched between Plex and PKC?

**Comment by croneter** [comment, marked as answer]
Watching Plex stuff in Kodi marked it as played on the Plex side, yes

**Reply by jaredcat** [reply]
hmm, dosent seem to work for me. Any ideas on troubleshooting?

**Reply by croneter** [reply]
Yes, share your DEBUG log file,see https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug
